.gitignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
This project doesn't vendor dependencies, if you need it by any reason please move it to `git config --global core.excludesfile ~/.gitignore_global` to avoid confusion why gitignore contains vendor dir.